sqladmin/pagination.py

```diff
@@ -52,6 +52,9 @@ def resize(self, page_size: int) -> Pagination:
         return self
 
     def add_pagination_urls(self, base_url: URL) -> None:
+        # Clamp page
+        self.page = min(self.page, max(1, self.count // self.page_size + 1))
+
```

Comment:
And I suggest to define this instead in the top:

```py
def __post_init__(self):
        # Clamp page
        self.page = min(self.page, max(1, self.count // self.page_size + 1))
```

Reply:
looks better, edited